srcpkgs/pipewire/patches/ncursesw.patch

```diff
@@ -0,0 +1,11 @@
+--- meson.build.orig	2021-01-05 18:40:24.345620177 +0100
++++ meson.build	2021-01-05 18:40:43.209619527 +0100
+@@ -325,7 +325,7 @@
+ pthread_lib = dependency('threads')
+ dbus_dep = dependency('dbus-1')
+ sdl_dep = dependency('sdl2', required : false)
+-ncurses_dep = dependency('ncurses', required : false)
++ncurses_dep = dependency('ncursesw', required : false)
```

Comment:
Is this necessary? I think our ncurses is always `ncursesw`.

Reply:
> Is this necessary? I think our ncurses is always `ncursesw`.

we have no `ncurses.pc`, only `ncursesw.pc`, so without the patch it is not found...

Reply:
Hmm, me neither... Should we have a `ncurses.pc` symlink, maybe?